frontend/app/routes.test.tsx

```diff
@@ -90,4 +90,22 @@ describe("routes", () => {
     // Test that we get the Not Found page
     await expectNotFound();
   });
+
+  test("Not found page when polling station is finalised", async () => {
+    const router = setupTestRouter();
+
+    // Navigate to a non-existing page
+    await router.navigate("/elections/1/data-entry/2");
+    expect(router.state.location.pathname).toEqual("/elections/1/data-entry/2");
+
+    // NOTE: We're not using the wrapped render function here,
+    // since we want control over our own memory router.
+    rtlRender(
+      <ApiProvider>
+        <RouterProvider router={router} />
+      </ApiProvider>,
+    );
```

Comment:
We can simplify this after my changes in https://github.com/kiesraad/abacus/pull/233/commits/cc2afed8e602cdca510e1f48ed850295d440f806:
```typescript
rtlRender(<Providers router={router} />);
```
Or to replace all invocations in this file:
```typescript
// top of file
const render = () => rtlRender(<Providers router={router} />);

// in the test
render();
```
and replace all duplicate code blocks.

Reply:
[72a87fe](https://github.com/kiesraad/abacus/pull/367/commits/72a87fee88145db3182e255116d1662337f4a3f0)